docs/pages/developer_installation.rst

```diff
@@ -0,0 +1,23 @@
+Developer installation and setup
+================================
+
+Test the installation
+---------------------
+
+If you'd like to run tests, make sure to install the package with the optional test dependencies. E.g.,
+
+.. code-block:: bash
+
+    pip install -e ".[test]"
+
+Then, in the tests directory, run
```

Comment:
"Then, in the `astrodb_utils/tests/` directory" -> we updated the path so we should make it clearer here